### PR TITLE
Set enableVolumeScheduling to true by default in the helm chart

### LINKED ISSUE #727: Helm chart should default enableVolumeScheduling: true
/kind bug

**What happened?**
Deployed the default helm chart and topology aware dynamic provisioning did not work. The kustomize based manifest default to true: https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/master/deploy/kubernetes/values/controller.yaml#L4

**What you expected to happen?**
With the CSI Topology sub-feature at GA status, I would expect this feature gate to default to true.

Thoughts on this?  Is there still a good reason to default the Helm chart to not use the Topology feature?

### PR DESCRIPTION
**Pull request by mtougeron**
**Is this a bug fix or adding new feature?**
Fixes #727 

**What is this PR about? / Why do we need it?**
Changes the default value of `enableVolumeScheduling` from `false` to `true` for the helm chart.

### CODE REVIEW

File: charts/aws-ebs-csi-driver/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: "0.9.0"
 name: aws-ebs-csi-driver
 description: A Helm chart for AWS EBS CSI Driver
-version: 0.9.6
+version: 0.9.7
```
Comment (ayberk) [inline charts/aws-ebs-csi-driver/Chart.yaml:5]: Can we update this to 0.9.8? We recently merged another change.

Comment (mtougeron) [reply charts/aws-ebs-csi-driver/Chart.yaml]: done

**Comment by ayberk** [comment]
/approve
/lgtm

Thanks!

**Pull request merged**